presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionTreeUtils.java

```diff
@@ -67,6 +72,11 @@ private static boolean isWindowFunction(FunctionCall functionCall)
         return functionCall.getWindow().isPresent();
     }
 
+    private static Predicate<FunctionCall> isExternalFunctionPredicate(Map<NodeRef<FunctionCall>, FunctionHandle> functionHandles, FunctionManager functionManager)
```

Comment:
This private helper method is only used once. Just inline?

Reply:
All the other predicates in this class are only used once. I'm just keeping them consistent (including the style) with the rest of the file.